pxtlib/browserutils.ts

```diff
@@ -851,4 +851,34 @@ namespace pxt.BrowserUtils {
             .then(db => db.clearAsync())
             .catch(e => deleteDbAsync().done());
     }
+
+    export interface IPointerEvents {
+        up: string,
+        down: string[],
+        move: string,
+        enter: string,
+        leave: string
+    }
+
+    export const pointerEvents: IPointerEvents = hasPointerEvents() ? {
+        up: "pointerup",
```

Comment:
This is fine for now, but let's look into having a util library that all these utils go to. 
webapp shouldn't need to rely on pxsim just for utils
pxtblocks shouldn't need to rely on pxtblocks just for utils..

you get the idea